common/eth2/src/lighthouse/attestation_performance.rs

```diff
@@ -0,0 +1,38 @@
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use types::Epoch;
+
+#[derive(Debug, Default, PartialEq, Clone, Serialize, Deserialize)]
+pub struct AttestationPerformanceStatistics {
+    pub active: bool,
+    pub head: bool,
+    pub target: bool,
+    pub source: bool,
+    pub delay: Option<u64>,
```

Comment:
let's add a `serde(skip_serializing_if = "Option::is_none")` to save outputting `null` values.